packages/nc-gui/helpers/parsers/parserHelpers.ts

```diff
@@ -191,3 +191,36 @@ export const filterNullOrUndefinedObjectProperties = <T extends Record<string, a
     return result
   }, {} as Record<string, any>) as T
 }
+
+/**
+ * Extracts the next default name based on the provided namesData, defaultName, and splitOperator.
+ *
+ * @param namesData - An array of strings containing existing names data.
+ * @param defaultName - The default name to extract and generate the next name from.
+ * @param splitOperator - The separator used to split the defaultName and numbers in existing namesData.
+ *                        Defaults to '-'. Example: If defaultName is 'Token' and splitOperator is '-',
+ *                        existing names like 'Token-1', 'Token-2', etc., will be considered.
+ * @returns The next default name with an incremented number based on existing namesData.
+ */
+export const extractNextDefaultName = (namesData: string[], defaultName: string, splitOperator: string = '-'): string => {
+  // Extract and sort numbers associated with the provided defaultName
+  const extractedSortedNumbers =
+    (namesData
+      .map((name) => {
```

Comment:
Array.prototype.map() expects a value to be returned at the end of arrow function